src/bootloaders/systemd-class.c

```diff
@@ -240,13 +242,23 @@ bool sd_class_install_kernel(const BootManager *manager, const Kernel *kernel)
                                  "linux %s/%s\n",
                                  get_kernel_destination_impl(manager),
                                  kernel->target.path);
+
         /* Optional initrd */
         if (kernel->target.initrd_path) {
                 cbm_writer_append_printf(writer,
                                          "initrd %s/%s\n",
                                          get_kernel_destination_impl(manager),
                                          kernel->target.initrd_path);
         }
+
+        boot_manager_initrd_iterator_init(manager, &iter);
+        while (boot_manager_get_initrd_freestanding_entry(&iter, initrd_name)) {
+                cbm_writer_append_printf(writer,
```

Comment:
I think discussed issuing a warning if some of the configuration entries will not be taken into account. Do you think it's not needed?

Reply:
every one of them will be taking into account if the initrds are using systemd as init